README.md

```diff
@@ -14,8 +14,8 @@
  See the License for the specific language governing permissions and
  limitations under the License.
 -->
-YAML SonarQube Plugin
-=====================
```

Comment:
I'll beg to differ a bit. Markdown headings style might not be relevant for display, but it should be consistent in the file. Only this H1 heading was with `====`, all other headings are using `#`. Therefore I would like to Keep this change.

Reply:
I agree but my intention is to mark document title, which is missing in MD syntax. So I'd like to keep those ===.